recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+openjdk:
+  - 8
+  - 9
+  - 10
```

Comment:
Why support jdk 9 and 10? They have reached end of support.

Reply:
Projects such as Imagej does not support Java 11 yet while it supports Java 9 which has modern feature Java 8 does not have.

Conda OpenJDK builds are available so I don't see anything bad to support it. Moreover, by default with `conda install openjdk`, people will get the latest Java 11 version.